demo-project/conf/base/catalog_01_raw.yml

```diff
@@ -3,18 +3,16 @@ companies:
   filepath: ${_base_location}/01_raw/companies.csv
   metadata:
     kedro-viz:
-      layer: raw
-      preview_args:
-        nrows: 5
+      preview: false 
+      
 
 reviews:
   type: pandas.CSVDataset
   filepath: ${_base_location}/01_raw/reviews.csv
   metadata:
     kedro-viz:
       layer: raw
-      preview_args:
-        nrows: 10
+      prvoew
```

Comment:
was this intentional change for testing ?

Reply:
this was a mistake :)  .. thanks for flagging